### DISCUSSION TITLE
Layout of the Mortgage calculator

### DISCUSSION THREAD

**Discussion by WeiZhou-dev** [Satish-Mutyala/T24S2T1EXTG12#10] [General]
I have finished the xmal code of my branch, please check if this is ok for everyone.
![2024-09-11 182628](https://github.com/user-attachments/assets/aec1cf90-0d0f-476f-9a15-dcac26ac9976)

**Comment by Satish-Mutyala** [comment]
Looks great @WeiZhou-dev 

Here is the screenshot of my converter.
![image](https://github.com/user-attachments/assets/b2d17693-10a5-4190-9ba8-468d342cb42b)
